include/configuration_server.hrl

```diff
@@ -0,0 +1,18 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%% This header file defines a few macros, that TEP needs to find and
+%%% access a configuration file.
```

Comment:
We know this is a header file from the file extension and contents. I'd like for us to avoid using "this x", "this y" repeatedly. Golang (as usual) has an opinion on this, which I actually find helpful in this case as a mental model. Their linter will mark comments as warn when they don't start with the name of the thing they're describing. 
Also: no comma before "that" in English.

Reply:
Also also: Shouldn't that fit on a single line?

Reply:
This will be changed shortly